appcenter-data/test/AppCenterPushMock.js

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Rename this file to **AppCenterDataMock.js**.

Reply:
Done